### DISCUSSION TITLE
Digi-Funktion geht bei mir nicht - Welche Einstellungen?

### DISCUSSION THREAD

**Discussion by dm2ov** [richonguzman/LoRa_APRS_iGate#170] [Q&A]
Ich möchte einen Digi auf einen Berg (ca. 2Km Entfernung) aufstellen und dann weiterleiten zu meinem iGate zu Hause, und dann in das APRS-Netz senden.
Meine Einstellungen  :  RX 433.775MHz aktiv , TX 433.775MHz aktiv , Digimode aktiv , Baken RF aktiv (sendet auch) , APRS-IS off .
Auf APRS. fi wird auch mein QTH angezeigt , Gerät ist TTGO LORa32 T3_V1.6.1 mit SW-V1.0.4.2 ,aber Digi funktioniert nicht,  (zu Hause als iGate OK!) .  Was mache ich Falsch...?

**Comment by richonguzman** [comment]
Hi

1) can you update to latest firmware update (1.0.5.2 currently)

2) can you post here the iGateBackup.json file to se your fully configuration?

**Reply by dm2ov** [reply]
Arbeite nur mit WEB FLASHER/INSTALLER  -  Mit V1.0.5.1 geht auch nicht...

**Reply by dm2ov** [reply]
Hier die Datei...
[iGateConfigurationBackup (2).json](https://github.com/user-attachments/files/16833266/iGateConfigurationBackup.2.json)

**Reply by Mane76** [reply]
> Hier die Datei... [iGateConfigurationBackup (2).json](https://github.com/user-attachments/files/16833266/iGateConfigurationBackup.2.json)

Die Datei schaut gut aus, die Sendeleistung ist stark reduziert eingestellt.
Woran machst Du fest dass die Digi Funktion nicht funktioniert ?

Mane

**Reply by dm2ov** [reply]
Wenn ich hinter dem Berg bin , hört mich ja das iGate nicht, und der Digi sendet nicht zu meinem igate ins aprsis.
Frage:(wie läuft der digi - empfängt RX und sendet nach ca. 5 sec. auf TX oder wie? 
Gruß wolfgang

**Reply by richonguzman** [reply]
does other igate listen your tracker where you are?

its common to thing the igate does not listen when other igates uploads your packets before the igate you want

**Comment by Mane76** [comment]
Hallo Wolfgang,

wie Ricardo schon schreibt, deine Konfiguration zu sehen wäre hilfreich.

Als Gedanke, für deinen Anwendungsfall bietet sich das "Split-Digipeaten" an, d.h der Digi am Berg empfängt auf der .775, sendet auf der .900 zu deinem iGate. Dein iGate muss dann auf der .900 empfangen und schickt das Paket zu aprsis. 
Der Vorteil ist, dass die .775 durch den Digi nicht belastet wird.

73
Mane

**Comment by richonguzman** [comment]
hi @dm2ov , as @Mane76 told before, the igateConfigurationBackup.json file looks good!

its nice you have 7 as power selected as you (maybe) dont need to reach that far with it.

now I need some info from you:

1) DM2OV-15 is sending at 10min rate (does it need to be that fast?). 15min to 30min rate is expected.

2)DM2OV-15 is being sending his gps beacons without any problem. So I need for you to explain as how you "don't see it" or is it that you dont see any repeated packet by it? You can check on received packets on your iGate to see if they appear there

**Reply by dm2ov** [reply]
Bei 20mW hören mich schon andere iGates.
Die 10min-Rate ist nur für  den Test.
Am iGate kommen keine pakete an.  Müßte doch schon in 1km Entfernung gehen, oder?

**Reply by richonguzman** [reply]
you dont see any packets on the igate that were repeated by the digirepeater board?

**Reply by dm2ov** [reply]
Ich sehe keine Pakete auf dem iGate vom digirepeater

**Reply by richonguzman** [reply]
are they on the same Freq and lora configuration?

is another igate listening to it to upload the digirepeater gps beacon packets?

**Comment by Mane76** [comment]
Hallo Wolfgang,

ein paar Worte zum Digi und zu dem was Ricardo meint. Ich glaube ich kenne den Grund deines Problems.

Der Digi kann auf zwei Arten betrieben werden, 1) RX und TX auf derselben Frequenz oder 2) RX und TX mit Offset.
Bei 1) werden nur Pakete gedigipeated, welche WIDE1-1 im Pfad haben. Bei Digipeaten selber wird dann das WIDE1-1 entfernt und somit das Paket kein zweites Mal gedigipeated. Das ist nötig um bei mehreren Digipeater kein Ping-Pong zu erzeugen und dadurch die Frequenz zu blockieren.
Bei 2) werden alle Pakete gedigipeatet.

Laut der Config steht dein Digipeater auf 1). Wenn ich mir deinen -11 Tracker anschaue versendet der kein WIDE1-1 im Pfad - somit wird das Paket nicht gedigipeatet. Für einen Test solltest Du beim Tracker WIDE1-1 in den Pfad einbauen, dann sollte das Paket auch gedigipeatet werden.

Wenn das dann geht kommt der zweite wichtige Punkt, den man im Zusammenhang mit APRS wissen muss:
Wenn ein Paket von mehreren iGates empfangen und zum aprsis geschickt werden, tritt eine Filterfunktion ein um Doubletten zu filtern. Es wird an aprs.fi nur das "schnellste iGate" weitergeleitet, die anderen werden verworfen. Beim Digi wird das Paket ca. 3 Sekunden nach Empfang gedigipeatet, d.h. es wird immer (!) das letzte Paket sein und somit verworfen werden. Ausser der Digi ist der einzige, der das Paket empfangen hat.

Als Zusammenfassung: 

- Bei Verwendung des Digi mit RX und TX auf der gleichen Frequenz werden nur Pakete mit WIDE1-1 im Pfad gedigipeatet
- Bei Verwendung des Digi mit getrennten RX und TX werden alle Pakate gedigipeatet. Du brauchst dann nur ein iGate das auf der TX Frequenz des Digis empfängt.
- Bei aprs.fi wird immer das "schnellste iGate" angezeigt, die anderen werden verworfen

Empfehlung:
- Bringe beim Tracker das WIDE1-1 in den Pfad. Ich sehe Du verwendest den APRSCube, da geht das bestimmt über die Config. Das hilft Dir dann auch bei anderen Digis wenn Du mobil unterwegs bist.
- Stelle den Digi auf getrennte RX und TX (RX .775, TX .900). Damit hältst Du die Frequenz frei
- Stelle dein iGate auf RX .900. Ich sehe Du hast auch hier den APRSCube, da geht das auch über die Config. Damit wird dein iGate nur (!) noch deinen Digi empfangen, aber Du hast nen sauberen Setup. Und wenn der Digi ne gute Position hat wirst Du damit mehr empfangen und der Community einen echten Mehrwert liefern.

73
Mane

**Reply by dm2ov** [reply]
Danke Danke für die ausführliche Beschreibung, und Deine Mühe.
Ich dachte ich könnte den digi einen weg eingeben - nur für mein igate , das er das paket nur meinenem igate sendet - , aber das geht warscheinlich mit getrentter RX-TX viel besser - nochmal danke. 
Werde getrente RX+TX verwenden, aber habe jetzt Gerüst am Haus stehen, wird etwas Zeit dauern.
Ich werde mich auf alle Fälle wieder melden, ob es geklappt hat (auch mit derConfig beim APRSCube)
73 DM2OV Wolfgang

**Reply by dm2ov** [reply]
Hallo mane76...
Habe alles versucht, auch dl3dcw angeschrieben, der aprscube unterstützt nicht Wide1-1, getrente QRG kann ich eingeben (QRG1+QRG2), funktioniert auch, aber wenn kein wide1-1 mit gesendet wird, was nicht vogesehen ist von dl3dcw, beim aprscube, wird auch kein paket vom digi weritergeleitet, also nur wahrseinlich zum IGate.
_Digi mit getrennten RX .775 und TX .900 werden alle Pakate gedigipeatet_ , das funtioniert bei mir nicht beim aprscube als iGate und aprscube als Tracker, muß ich da kompett auf ricardo als igate umsteigen??
Mein Probe aufbau:
( -14 ist iGate aprscube RX .900 und TX .900 / -15 ist ricardo digi RX .775 und TX .900 / habe testweise -13  ricardo digi mit gleicher QRG TX=RX .775 geht aber auch nicht ) APRScube wird einfach nicht gedigipeatet...
Also wenn man mit APRScube ( der Würfel Kompackt) an einen DIGI vorbeifährt, wird man nicht weitergeleitet zu aprs.fi  .
Gruß DM2OV

**Comment by Mane76** [comment]
Hallo Wolfgang,

ok, das ist dann ungünstig wenn beim Cube der Pfad nicht angepasst werden kann.
Zu Deinem Setup:

Der -15 ist der Split-Frequenz-Digi von Ricardo, richtig (RX .775, TX .900) ? 
Ich sehe dass dieser alle 10min eine Bake sendet und diese Bake wird vom -14 (dann auf .900) empfangen. Somit funktioniert schonmal der Weg vom Digi zum iGate.
In den Einstellungen vom -15, hast Du da in der Sektion "Digipeating" den  Mode auf "WIDE1-1" stehen ? Nur um sicher zu gehen.

Welches Gerät ist denn aktuell der -13 ? Das ist ein Gerät mit Ricardos Software, welches über den -15 gedigipeatet wird und dann vom -14 empangen wird. Also hier funktioniert die Kette. Ich nehme an das läuft nur zum Test, ansonsten würde es keinen Sinn machen.

Dann hast Du ja noch einen Cube als Tracker, oder ? -11 und -9... die senden nur auf der .775, oder ?

Grüße
Mane

**Reply by dm2ov** [reply]
Hallo Mane...
Ja der -15 ist der Split Digi, und Digipeating steht auf WIDE1-1.
Und der -13 ist zum Test (RX .775 +TX .775 + "Digipeating" den Mode auf "WIDE2+WIDE1-1") , ob die Weiterleitung zur -14 geht.
Der Cube -11 mit TX .775  und der Cube -9 aktuell TX -900 zu Cube-14 zum iGate.
Nur der Cube -11 wird nicht zum iGate geleitet.
-(_Bei Verwendung des Digi mit getrennten RX und TX werden alle Pakate gedigipeatet. Du brauchst dann nur ein iGate das auf der TX Frequenz des Digis empfängt_.) Deiner Aussage nach , geht diese Einstellung bei mir noch nicht, oder ist nicht so programiert.
Gruß DM2OV

**Reply by Mane76** [reply]
Hallo Wolfgang, 

Habs grad bei mir nachgestellt und kann bestätigen dass ein Fehler vorliegt. Bei Split wird nicht alles gedigipeatet.

Ricardo schaut sich das an, melde mich wenn es wieder funktioniert. 

Grüße

**Comment by Mane76** [comment]
Hallo Wolfgang, das Problem ist gelöst, Split-Digipeating funktioniert wieder.
Kannst Du die aktuelle Version von Ricardo mittels VSCode installieren ?

Grüße

**Reply by dm2ov** [reply]
Hallo mane...
Nein mit VSCode kann ich das nicht, habe ich noch nie gemacht. 
Wo bekommt man die .bin Datei her. Arbeite nur mit Flash_Download_Tool_v3.8.5.zip oder dem WebFlasher.
Wenn nicht muß ich halt auf ein neues Update von Ricardo warten, oder einen anderen OM in meiner Nähe fragen, um mir das zu flashen.
Soweit bin ich noch nicht, das ich VSCode habe und nutze.
Gruß DM2OV

**Comment by Mane76** [comment]
Hallo Wolfgang,
Ricardo hat das Feature in den Webupdater integriert, Du kannst testen.

'73

**Reply by richonguzman** [reply]
all was added to WebFlasher in iGate (V2.0.1)

**Reply by dm2ov** [reply]
Habe mit WebFlasher die neue Version installiert. 
Geht ausgezeichnet, auch den ECOMode getestet - das ist ja sehr gut, nur noch 25mA Stromverbrauch. 
Werde demnächst den DIGI auf den Berg aufbauen, und weiter testen...
Besten Dank an Ricardo für das schnelle integrieren.
73 DM2OV

**Comment by richonguzman** [comment]
Great News let me know how it goes ;)
